src/null-safety/faq.md

```diff
@@ -175,17 +175,17 @@ An explicit runtime null check, for example `if (arg == null) throw
 ArgumentError(...)`, will be flagged as an unnecessary comparison if you make
 `arg` non-nullable.
 
-But, the check *is* still needed when running in mixed mode. Until everything is
-fully migrated and the code switches to running in sound mode, it will be
+But, the check *is* still needed when running with mixed versions. Until everything is
```

Comment:
"mixed versions" is a property of the program, not how you run it, so let's do something like:

"when running with mixed versions" -> "if the program is a mixed-version one".